### PR TITLE
Besca issue#3

### PR DESCRIPTION
**Pull request by kohleman**
Hi Klas, can you have a look at this pull request?

### CODE REVIEW

File: besca/st/__init__.py [hunk 1]
```diff
@@ -21,6 +21,7 @@
     pca_neighbors_umap,
     clustering,
     additional_labeling,
+    additional_labeling,
```
Comment (hatjek) [inline besca/st/__init__.py:24]: additional_labeling seems to be duplicated in besca/st/__init__.py.

File: besca/st/__init__.py [hunk 2]
```diff
@@ -39,4 +40,5 @@
     "export_celltype",
     "additional_labeling",
     "celltype_labeling",
+    "additional_labeling",
```
Comment (hatjek) [inline besca/st/__init__.py:43]: Same here.

**Review by hatjek — COMMENTED** [review]
There seems to be some duplication of "additional_labeling" in besca/st/__init__.py.

**Comment by hatjek** [comment]
@yannik-benndorf Could you maybe have a look into the pytest changes? It's difficult for me to judge.

**Comment by yannik-benndorf** [comment]
@hatjek doctest and pytest look good to me.

**Review by hatjek — APPROVED** [review]

**Pull request merged**